include/MainWindow.h

```diff
@@ -30,6 +30,7 @@
 #include <QtCore/QList>
 #include <QMainWindow>
 
+#include "ConfigManager.h"
```

Comment:
I don't remember why this moved... I think it was when I had problems compiling ( earlier in this thread ).
Moved back.